### PR TITLE
Unit test

### PR DESCRIPTION
**Pull request by firoouzeh**

### CODE REVIEW

File: test/Unit/AESCryptoTest.php [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+use PHPUnit\Framework\TestCase;
+
+class AESCryptoTest extends TestCase
+{
+
+    public function testReturnTypeValidatorForSecureNoteRequest()
+    {
+        $crypto = new AESCrypto();
+        $secureNoteResult = $crypto->secureNoteRequest("this is a test note");
+        $this->assertInstanceOf(SecureNote::class,$secureNoteResult);
```
Comment (ph4r5h4d) [inline test/Unit/AESCryptoTest.php]: Kindly change it to `assertNotNull` this would suffice.

File: test/Unit/AESCryptoTest.php [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    }
+
+    public function testIvAndKeyLengthInSecureNoteRequest()
+    {
+        $crypto = new AESCrypto();
+        $secureNoteResult = $crypto->secureNoteRequest("this is a test note");
+
+        $lengthOfIv = strlen($secureNoteResult->getIv());
+        $this->assertEquals(16,$lengthOfIv);
```
Comment (ph4r5h4d) [inline test/Unit/AESCryptoTest.php]: follow code styles, `$this->assertEquals(16,$lengthOfIv);` should be `$this->assertEquals(16, $lengthOfIv);` for example.

**Review by ph4r5h4d — CHANGES_REQUESTED** [review]

File: test/Unit/AESCryptoTest.php (hunk 1 above)
Comment (firoouzeh) [reply test/Unit/AESCryptoTest.php]: it's done

File: test/Unit/AESCryptoTest.php (hunk 2 above)
Comment (firoouzeh) [reply test/Unit/AESCryptoTest.php]: it's done

**Pull request merged**